mkosi/config.py

```diff
@@ -2308,9 +2322,10 @@ def line_join_source_target_list(array: Sequence[tuple[Path, Optional[Path]]]) -
 
 
 def summary(args: MkosiArgs, config: MkosiConfig) -> str:
-    b = Style.bold
-    e = Style.reset
-    bold: Callable[..., str] = lambda s: f"{b}{s}{e}"
+    def bold(s: Any) -> str:
+        b = Style.bold
+        e = Style.reset
```

Comment:
Is there any point in still having these at all? just inline them?

Reply:
Will change it like that.